samples/android/BeaconServiceDemoApp/app/src/main/java/com/google/sample/beaconservice/MainActivityFragment.java

```diff
@@ -245,6 +245,19 @@ private void createScanner() {
   }
 
   @Override
+  public void onResume() {
+    super.onResume();
+
+    // There could be multiple instances when we need to handle 'UserRecoverableAuthExceptions'.
+    // (i.e to updated GMS, to re-login, for permission to use the proximity beacon api)
+    // So run this check every time another activity has finished running.
+    String accountName;
+    if ((accountName = getActivity().getSharedPreferences(Constants.PREFS_NAME, 0).getString("accountName", "")) != "") {
```

Comment:
Oh, yep.... opps. I've been working with Go a lot lately and must of mixed things up. Thanks :)